FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -267,32 +309,38 @@ if (( generate_template )); then
 	  recon_all_cmd+=" -norandomness -rng-seed ${recon_all_seed}"
 	fi
 
-
 	#---------------------------------------------------------------------------------------
 	log_Msg "Running the recon-all to generate common template"
 	#---------------------------------------------------------------------------------------
 
-	#recon_all_cmd+=(${extra_reconall_args_base[@]+"${extra_reconall_args_base[@]}"})  
+	#recon_all_cmd+=(${extra_reconall_args_base[@]+"${extra_reconall_args_base[@]}"})
 	recon_all_cmd+=" $extra_reconall_args_base "
 	echo "recon_all_cmd:"
 	echo ${recon_all_cmd}
 	log_Msg "...recon_all_cmd: ${recon_all_cmd}"
 
-	${recon_all_cmd}
-	return_code=$?
-	if [ "${return_code}" != "0" ]; then
+  if [ $PARALLEL_MODE == "FSLSUB" ]; then 
+    template_job=$(${recon_all_cmd[*]})
+    par_waitjobs_fslsub $template_job
+  else
+    ${recon_all_cmd[*]} 
+    par_runjobs 1         #In NONE mode, returns return code of the previous command
+  fi
```

Comment:
This is silly.  There is basically no point to using job submission to submit one job and then wait for it to finish.  If there is nothing to do in parallel, just run the real command in the context of this script.

Reply:
I guess if this were a batch script, and not the pipeline script, then it would make sense to submit and wait for even a single job (but I thought fsl_sub -j handled multiple job ids without making the launcher script wait?), but that should be in `Examples/Scripts`, and probably shouldn't have the symlinking stuff in it.